backend/oas/user/schemas/devices.yaml

```diff
@@ -50,80 +48,17 @@ devices.DeviceInfo:
       type: string
       example: "State vector-based quantum circuit simulator"
   required: [
-    deviceId, deviceType, status, nPendingTasks, basisGates, supportedInstructions, description
+    deviceId, deviceType, status, nPendingJobs, basisGates, supportedInstructions, description
```

Comment:
should be snake case.

`deviceId, deviceType, status, nPendingJobs, basisGates, supportedInstructions, description`

Reply:
Finished to modify.